plugins/hive-plugin/src/test/java/edu/berkeley/ground/plugins/hive/TestGroundStore.java

```diff
@@ -0,0 +1,374 @@
+        binaryData.setNumNulls(nulls);
+        data.setBinaryStats(binaryData);
+        colStatsObj.setStatsData(data);
+        return colStatsObj;
+    }
+
+    private static ColumnStatisticsObj mockDecimalStats(int i) {
+        Decimal high = new Decimal();
+        high.setScale((short) 3);
+        String strHigh = String.valueOf(3876 + 100 * i);
+        high.setUnscaled(strHigh.getBytes());
+        Decimal low = new Decimal();
+        low.setScale((short) 3);
+        String strLow = String.valueOf(38 + i);
+        low.setUnscaled(strLow.getBytes());
+        long nulls = 13 + i;
+        long dVs = 923947293L + 100 * i;
+        ColumnStatisticsObj colStatsObj = new ColumnStatisticsObj();
+        colStatsObj.setColName(DECIMAL_COL);
+        colStatsObj.setColType(DECIMAL_TYPE);
+        ColumnStatisticsData data = new ColumnStatisticsData();
+        DecimalColumnStatsData decimalData = new DecimalColumnStatsData();
+        decimalData.setHighValue(high);
+        decimalData.setLowValue(low);
+        decimalData.setNumNulls(nulls);
+        decimalData.setNumDVs(dVs);
+        data.setDecimalStats(decimalData);
+        colStatsObj.setStatsData(data);
+        return colStatsObj;
+    }
+
+    @AfterClass
+    public static void afterTest() {
+    }
+
+    @Before
+    public void init() throws IOException {
+        MockitoAnnotations.initMocks(this);
+        HiveConf conf = new HiveConf();
+        conf.setBoolean(GroundReadWrite.NO_CACHE_CONF, true);
+        store = MockUtils.init(conf);
+    }
+
+    @Test
+    public void createDb() throws Exception {
+        String dbname = "mydb";
+        Database db = new Database(dbname, "no description", "file:///tmp", emptyParameters);
+        store.createDatabase(db);
+
+        Database d = store.getDatabase(dbname);
+        Assert.assertEquals(dbname, d.getName());
+        Assert.assertEquals("no description", d.getDescription());
+        Assert.assertEquals("file:///tmp", d.getLocationUri());
+    }
+
+    @Test
+    public void createTable() throws Exception {
+        String tableName = "mytable";
+        int startTime = (int) (System.currentTimeMillis() / 1000);
+        List<FieldSchema> cols = new ArrayList<FieldSchema>();
+        cols.add(new FieldSchema("col1", "int", ""));
+        SerDeInfo serde = new SerDeInfo("serde", "seriallib", null);
+        Map<String, String> params = new HashMap<String, String>();
+        params.put("key", "value");
+        StorageDescriptor sd = new StorageDescriptor(cols, "file:/tmp", "input", "output", false, 17, serde,
+                Arrays.asList("bucketcol"), Arrays.asList(new Order("sortcol", 1)), params);
+        Table table = new Table(tableName, "default", "me", startTime, startTime, 0, sd, null, emptyParameters, null,
+                null, null);
+        store.createTable(table);
+
+        Table t = store.getTable("default", tableName);
+        Assert.assertEquals(1, t.getSd().getColsSize());
```

Comment:
Can we static import `Assert`, so we don't have to do `Assert.` before every assertion?